lib/validator.js

```diff
@@ -60,9 +58,10 @@ const lintObject = (objectName, object, options) => {
 }
 
 function validateUrl(s, contextServers, context, options) {
-    s.should.be.a.String().and.not.be.Null();
+    s.should.be.a.String();
+    s.should.not.be.Null();
     if (!options.laxurls) s.should.not.be.exactly('', 'Invalid empty URL ' + context);
-    var base = options.origin || 'http://localhost/';
+    let base = options.origin || 'http://localhost/';
     if (contextServers && contextServers.length) {
         const servers = contextServers[0];
         if (servers.length) {
```

Comment:
Again - I think this is the actual culprit. Suggest changing to `if (servers && servers.length) {`.